build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
```

Comment:
For bonus points, add this at the top of any bash script:

```bash
set -ex
```

(see https://www.peterbe.com/plog/set-ex)